### ISSUE TITLE
Add FakeBackendV2 for Tokyo, Tenerife and Rueschlikon

### ISSUE THREAD

**Issue by HuangJunye** [Qiskit/qiskit-terra#7759]
### What should we add?

https://github.com/Qiskit/qiskit-terra/pull/7643 implemented mocked IBM Quantum Systems backends using the new BackendV2 interface. 

However, FakeBackendV2 class was not implemented for Tokyo, Tenerife and Rueschlikon quantum systems because there is no configuration file exist for these old systems.

## Suggested solution
Create fake conf files that specify key specifications of these devices based on other conf files, for example [quito](https://github.com/Qiskit/qiskit-terra/blob/main/qiskit/test/mock/backends/quito/conf_quito.json).

**Comment by jakelishman** [comment]
These three are all super old and retired - I don't think anybody has any configurations for them any more, and I think we'll be fine leaving them behind for `BackendV2`.  See also #7686.

**Comment by HuangJunye** [comment]
Yeah I agree with you. @mtreinish What do you think?

**Comment by mtreinish** [comment]
Yeah, I think it's fine to just exclude them. The further back in time the less important having fake backends are. We don't really need to test how things perform against a Rueschlikon like backend anymore using new interfaces because we'll never run with the new interfaces against Rueschlikon.

I'm going to close this as it's not feasible (if it was we'd have already done it for BackendV1's fake backends)
